input/pages/measure-repository-service.md

```diff
@@ -291,17 +291,18 @@ The [CQFMPublishableMeasureRepository](CapabilityStatement-publishable-measure-r
 
 A PublishableMeasureRepository:
 
-1. SHALL support library packaging: [Library/$package](OperationDefinition-Library-package.html) operation
+1. SHALL support library packaging: [Library/$qm.package](OperationDefinition-qmig-package.html) operation
     1. SHALL support the url parameter
     2. SHALL support the version parameter
     3. SHOULD support the offset parameter
     4. SHOULD support the count parameter
-    5. SHOULD support system-version parameter (overrides code system versions specified in the manifest)
-    6. SHOULD support check-system-version parameter (overrides code system versions specified in the manifest)
-    7. SHOULD support force-system-version parameter (overrides code system versions specified in the manifest)
+    5. SHOULD support canonicalVersion parameter (overrides code system versions specified in the manifest)
```

Comment:
Another note here, we're not replacing these parameters, we're adding the canonical ones, so we should still recommend support for these parameters.